Dockerfile

```diff
@@ -16,7 +16,9 @@ RUN yarn test
 RUN yarn build
 
 # setup Python
-FROM python:3.11.0-alpine3.15 AS backend
+# FIXME: do not use alpine, as aline does not
```

Comment:
```suggestion
# NOTE: do not use alpine, as alpine does not
```

Reply:
what is the problem with building from source? except that it takes a bit more time?
also do we share now binaries between a glibc and a musl system?

Reply:
It doesn't work thats the problem. Some dependencies build process is broken with this setup